ddtrace/internal/native/__init__.py

```diff
@@ -21,39 +17,20 @@ def get_configuration_from_disk(
     """
     configurator = PyConfigurator(debug_logs)
 
-    # Set the file override if provided. Only used for testing purposes.
+    # Check if the file override is provided via environment variables
+    # This is only used for testing purposes
+    local_file_override = os.getenv("_DD_SC_LOCAL_FILE_OVERRIDE", "")
+    managed_file_override = os.getenv("_DD_SC_MANAGED_FILE_OVERRIDE", "")
     if local_file_override:
         configurator.set_local_file_override(local_file_override)
     if managed_file_override:
         configurator.set_managed_file_override(managed_file_override)
 
-    config = []
+    config = {}
     try:
-        config = configurator.get_configuration()
+        config = {entry["name"]: entry for entry in configurator.get_configuration()}
```

Comment:
@BaptisteFoy Can we store the stable configuration values in a map? 

Are there any valid scenarios where the `PyConfigurator` returns two more entries for the same environment variable?

Reply:
Yes, if the environment variable is defined in both the local file & the fleet-managed file